drivers/hmis/app/models/hmis/unit_type.rb

```diff
@@ -6,42 +6,16 @@
 
 module Hmis
   class UnitType < HmisBase
-
-    # https://docs.google.com/spreadsheets/d/1xuXIohyPguAw10KcqlqiF23qgbNzKvAR/edit#gid=844425140
```

Comment:
@ttoomey the unit types listed here actually represent the Unit Types that should be available in the system. So, I added a JSON file and seed function to load them.